### PR TITLE
Add PAE LargeAdressAware for 32bit use up to 4GB RAM on Windows 64bit

### PR DESCRIPTION
**Pull request by informatorius**
Solves issue https://github.com/notepad-plus-plus/notepad-plus-plus/issues/681

### CODE REVIEW

**Comment by NN---** [comment]
You should change notepadPlus.vcxproj as well.

File: PowerEditor/visual.net/notepadPlus.vs2015.vcxproj [hunk 1]
```diff
@@ -117,6 +117,7 @@
       <ProgramDatabaseFile>$(OutDir)notepadPlus.pdb</ProgramDatabaseFile>
       <SubSystem>Windows</SubSystem>
       <TargetMachine>MachineX86</TargetMachine>
+	  <LargeAddressAware>true</LargeAddressAware>
```
Comment (NN---) [inline PowerEditor/visual.net/notepadPlus.vs2015.vcxproj]: Did you change the file in VS ?
Seems indentation is wrong.

**Comment by informatorius** [comment]
Can you change these findings yourself when merging? Or should I make a new pull request?

**Comment by NN---** [comment]
I don't have commit permissions.
You can update this PR.
Just make another commit to the PR branch and it will appear automatically.

**Comment by informatorius** [comment]
Review comments implemented

**Comment by donho** [comment]
It's not necessary since we've got 64-bit now.

**Comment by informatorius** [comment]
Could still be usefull for 32bit version as long as plugins are not supporting x64

**Pull request closed without merging**